x-pack/legacy/plugins/monitoring/public/components/chart/horizontal_legend.js

```diff
@@ -61,38 +57,37 @@ export class HorizontalLegend extends React.Component {
   }
 
   createSeries(row, rowIdx) {
-    const classes = ['col-md-4 col-xs-6 monRhythmChart__legendItem'];
+    const classes = ['monRhythmChart__legendItem'];
 
     if (!includes(this.props.seriesFilter, row.id)) {
       classes.push('monRhythmChart__legendItem-isDisabled');
     }
     if (!row.label || row.legend === false) {
-      return (
-        <div
-          key={rowIdx}
-          style={{ display: 'none' }}
-        />
-      );
+      return <div key={rowIdx} style={{ display: 'none' }} />;
     }
 
     return (
       <EuiKeyboardAccessible key={rowIdx}>
-        <div
+        <EuiFlexItem
+          grow={false}
           className={classes.join(' ')}
           onClick={event => this.props.onToggle(event, row.id)}
```

Comment:
I haven't looked at how this looks in the browser yet, but I think you want your onClick event on something other than an `EuiFlexItem` which is a presentation element. Again, not super familiar with this layout, but something like...

```
<EuiFlexItem><button onClick={}>Content of thing that needs to be pressed</button><EuiFlexItem>
```